basket/petition/tests/test_views.py

```diff
@@ -98,6 +98,27 @@ def test_petition_email_invalid(client, mocker):
     assert mock_send_mail.call_count == 0
 
 
+@pytest.mark.django_db
+def test_petition_db_error(client, mocker):
+    mock_send_mail = mocker.patch("basket.petition.models.send_email_confirmation")
+    url = reverse("sign-petition")
+    data = {
+        "name": "The Dude 😎",
+        "email": "thedude@example.com",
+        "title": "Dude",
+        "affiliation": "The Knudsens",
+    }
+    response = client.post(url, data=data)
+    assert response.status_code == 200
+    assert response.json() == {
+        "status": "error",
+        "errors": {
+            "__all__": "Database error",
```

Comment:
Is this raised because of the emoji? Just wondering how it's triggered.

Reply:
Yeah an emoji will trigger it, or most any other database error, but the emoji is an easy test.